src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -561,7 +561,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import '../../assets/buttons';
+@import '../../assets/variables';
```

Comment:
Was indirectly imported before and is needed to avoid:
```
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Undefined variable.
    ╷
609 │             opacity: $opacity_full;
    │                      ^^^^^^^^^^^^^
```